be/test/block_cache/test_cache_utils.h

```diff
@@ -0,0 +1,68 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <fmt/format.h>
+
+#include "block_cache/block_cache.h"
+#include "common/logging.h"
+
+namespace starrocks {
+
+template <typename T>
+class ScopedConfigUpdater {
+public:
+    ScopedConfigUpdater(T& param, T value) : _param(param), _saved_old_value(param) { param = value; }
+    ~ScopedConfigUpdater() { _param = _saved_old_value; }
+
+private:
+    T& _param;
+    T _saved_old_value;
+};
+
```

Comment:
move this to a separate file, expected to be used widely among all UTs.

Reply:
> move this to a separate file, expected to be used widely among all UTs.

Move it to `src/testutil/scoped_updater.h`.